packages/nodes-base/nodes/Schedule/ScheduleTrigger.node.ts

```diff
@@ -485,10 +511,23 @@ export class ScheduleTrigger implements INodeType {
 			}
 
 			if (interval[i].field === 'hours') {
-				const hour = interval[i].hoursInterval?.toString() as string;
+				const hour = interval[i].hoursInterval as number;
 				const minute = interval[i].triggerAtMinute?.toString() as string;
 				const cronTimes: string[] = [minute, `*/${hour}`, '*', '*', '*'];
 				const cronExpression: string = cronTimes.join(' ');
+				if (hour === 1) {
```

Comment:
I think we're now relying too much on the Cron lib and for an execution that should happen every X hours the `setInterval` would be a better approach, WDYT?